helm_deploy/send-legal-mail-to-prisons/values.yaml

```diff
@@ -40,6 +40,8 @@ generic-service:
       SYSTEM_CLIENT_ID: "SYSTEM_CLIENT_ID"
       SYSTEM_CLIENT_SECRET: "SYSTEM_CLIENT_SECRET"
       SESSION_SECRET: "SESSION_SECRET"
+      SLM_TAG_MANAGER_CONTAINER_ID: "SLM_TAG_MANAGER_CONTAINER_ID"
```

Comment:
As discussed in standup this morning we will need 2 Google Tag Manager container IDs - one for the legal sender journey (SLM), and one for the mail room journey (Check Rule 39)

At the moment we only have one container setup for SLM which is associated with the 3 send-legal-mail domains (dev, preprod and prod); therefore I have only setup the new secret for `SLM_TAG_MANAGER_CONTAINER_ID` (the same value in all 3 namespaces)

SLM-120 is to get GA working for the Mail Room journey, and it wont be much more than creating and configuring the container in GA/GTM, getting the container ID, creating a new secret CHECK_RULE39_MAIL_TAG_MANAGER_CONTAINER_ID in all 3 namespaces with that new container ID, and uncommenting out this line